srcpkgs/rawtherapee/template

```diff
@@ -1,20 +1,20 @@
 # Template file for 'rawtherapee'
 pkgname=rawtherapee
-version=5.10
-revision=3
+version=5.11
+revision=1
 build_style=cmake
-configure_args="-DCACHE_NAME_SUFFIX=\"\" -DWITH_LTO=ON"
+configure_args="-DCACHE_NAME_SUFFIX=\"\" -DWITH_LTO=ON -DWITH_SYSTEM_LIBRAW=ON"
 hostmakedepends="pkg-config"
-makedepends="fftw-devel gtkmm-devel lensfun-devel
- libcanberra-devel libgomp-devel libiptcdata-devel librsvg-devel exiv2-devel"
+makedepends="fftw-devel gtkmm-devel lensfun-devel libcanberra-devel libgomp-devel
+ libiptcdata-devel librsvg-devel exiv2-devel libraw-devel libjxl-devel"
 depends="desktop-file-utils hicolor-icon-theme"
 short_desc="Free RAW converter and digital photo processing software"
 maintainer="Daniel Martinez <danielmartinez@cock.li>"
 license="GPL-3.0-or-later"
 homepage="https://rawtherapee.com/"
-changelog="https://github.com/Beep6581/RawTherapee/blob/${version}/RELEASE_NOTES.txt"
-distfiles="https://rawtherapee.com/shared/source/rawtherapee-${version}.tar.xz"
-checksum=a799b53cd54dba4a211479e342ffc9c5db1c44d3d6c3a27d5cc13adf0debd2da
+changelog="https://raw.githubusercontent.com/Beep6581/RawTherapee/${version}/RELEASE_NOTES.txt"
```

Comment:
changelog uses `${version}` since the changelog is different between the default `dev` branch and the release branch